cs-property-service/src/main/java/org/egov/cpt/web/controllers/DemandGenerationController.java

```diff
@@ -0,0 +1,42 @@
+package org.egov.cpt.web.controllers;
+
+import javax.validation.Valid;
+
+import org.egov.cpt.models.DemandCriteria;
+import org.egov.cpt.models.RequestInfoWrapper;
+import org.egov.cpt.service.DemandGenerationService;
+import org.egov.cpt.util.ResponseInfoFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/demand")
+public class DemandGenerationController {
```

Comment:
Rename to `RentDemandGenerationController`.
Also the request mapping should be `/rent-demands` and method is `/_create`

Reply:
names changed as per recommendation